test/lint/lint.js

```diff
@@ -24,12 +21,30 @@ cli.addPlugin('eslint-plugin-unicorn', unicorn);
 
 const report = cli.executeOnFiles(files);
 
-if (fix) {
+const {errorCount, warningCount, fixableErrorCount, fixableWarningCount} = report;
+
+const hasFixable = fixableErrorCount || fixableWarningCount;
+
+if (fix && hasFixable) {
 	CLIEngine.outputFixes(report);
 }
 
-const formatter = cli.getFormatter();
+if (errorCount || warningCount) {
+	const formatter = cli.getFormatter();
+	console.log(formatter(report.results));
 
-console.log(formatter(report.results));
+	console.log();
+	console.log('Some test fails, you need fix them, and run `npm run lint <file>` again.');
```

Comment:
```suggestion
	console.log('Some test have failed, to fix them run `npm run lint <file>` again.');
```

Reply:
`npm run lint <file>` can't fix them, need fix and run to check

Reply:
Would `to what needs to be fixed`... Be better?

Reply:
noop, what needs to fix is already listed.

how about 

> Some tests have failed, you need fix them and run `npm run lint <file>` to check again.

Reply:
@fisker aha! Yes that sounds good. (I misunderstood what the message was supposed to say 😂)